migrations/tables/instance-report.yaml

```diff
@@ -37,6 +37,8 @@ spec:
         type: text
       - name: embedded_cluster_id
         type: text
+      - name: embedded_cluster_version
+        type: text
```

Comment:
i think this table is not used. won't hurt to add this, but the table is generally outdated now (some missing datapoints). should we consider removing it?